docs/schema.md

```diff
@@ -347,6 +353,27 @@ eg.
           unique: false
 ```
 
+## Indexes
+
+Developer can specify which on which columns index should be added.
+
+Each index has name and following options:
+
+- columns
+    List of column names to be indexed
+- type (optional)
+    Index type, available options are: "spatial", "fulltext", "unique"
```

Comment:
not true for SQLite, which supports "unique" and "" (https://sqlite.org/lang_createindex.html)